reconcile/utils/saasherder/models.py

```diff
@@ -191,6 +195,7 @@ class TargetSpec:
     cluster: str
     namespace: str
     managed_resource_types: Iterable[str]
+    managed_resource_names: Optional[list[ManagedResourceName]]
```

Comment:
And convert this class to a pydantic model:

```
class TargetSpec(BaseModel):
```

for a very strange reason you can't assign a `Sequence` to a `list` with a `dataclass` :(

Reply:
https://github.com/app-sre/qontract-reconcile/pull/3814/commits/ae8ecec604dc05b1544f73f920a29598df19260d